src/rewrite_infinite_loops.cpp

```diff
@@ -0,0 +1,316 @@
+/**
+ * @file rewrite_infinite_loops.cpp
+ *
+ * @author  Alex Henrie
+ * @license GPL v2+
+ */
+
+#include "rewrite_infinite_loops.h"
+
+#include "chunk.h"
+#include "uncrustify.h"
+
+
+using namespace uncrustify;
+
+
+static Chunk *find_end_brace(Chunk *pc)
+{
+   while (!pc->IsBraceOpen())
+   {
+      pc = pc->GetNextNcNnl();
+   }
+   return(pc->SkipToMatch());
+}
+
+
+static bool for_needs_rewrite(Chunk *pc, E_Token desired_type)
+{
+   // The 'for' statement needs to be rewritten if `for(;;)` is not the
+   // preferred syntax for infinite loops and this 'for' is an infinite loop
+   // with no extra tokens (such as inline comments).
+
+   if (desired_type == CT_FOR)
+   {
+      return(false);
+   }
+   pc = pc->GetNext();
+
+   if (!chunk_is_token(pc, CT_SPAREN_OPEN))
```

Comment:
you can replace ```chunk_is_token()``` with ```->Is()``` since we have started to do this in the code.

Reply:
Done